storybooknative/src/Button/index.js

```diff
@@ -1,13 +1,176 @@
+  const tokens = {
+    [TOKENS.backgroundButton]: {
+      [TYPE_OPTIONS.PRIMARY]: theme.orbit.backgroundButtonPrimary,
+      [TYPE_OPTIONS.SECONDARY]: theme.orbit.backgroundButtonSecondary,
+      [TYPE_OPTIONS.INFO]: theme.orbit.backgroundButtonInfo,
+      [TYPE_OPTIONS.SUCCESS]: theme.orbit.backgroundButtonSuccess,
+      [TYPE_OPTIONS.WARNING]: theme.orbit.backgroundButtonWarning,
+      [TYPE_OPTIONS.CRITICAL]: theme.orbit.backgroundButtonCritical,
+      [TYPE_OPTIONS.FACEBOOK]: theme.orbit.backgroundButtonFacebook,
+      [TYPE_OPTIONS.GOOGLE]: theme.orbit.backgroundButtonGoogle,
+    },
+    [TOKENS.borderColorButton]: {
+      [TYPE_OPTIONS.PRIMARY]: theme.orbit.borderColorButtonPrimaryBordered,
+      [TYPE_OPTIONS.SECONDARY]: theme.orbit.borderColorButtonSecondaryBordered,
+      [TYPE_OPTIONS.INFO]: theme.orbit.borderColorButtonInfoBordered,
+      [TYPE_OPTIONS.SUCCESS]: theme.orbit.borderColorButtonSuccessBordered,
+      [TYPE_OPTIONS.WARNING]: theme.orbit.borderColorButtonWarningBordered,
+      [TYPE_OPTIONS.CRITICAL]: theme.orbit.borderColorButtonCriticalBordered,
+      [TYPE_OPTIONS.FACEBOOK]: theme.orbit.borderColorButtonFacebookBordered,
+      [TYPE_OPTIONS.GOOGLE]: theme.orbit.borderColorButtonGoogleBordered,
+    },
+    [TOKENS.colorTextButton]: {
+      [TYPE_OPTIONS.PRIMARY]: theme.orbit.colorTextButtonPrimary,
+      [TYPE_OPTIONS.SECONDARY]: theme.orbit.colorTextButtonSecondary,
+      [TYPE_OPTIONS.INFO]: theme.orbit.colorTextButtonInfo,
+      [TYPE_OPTIONS.SUCCESS]: theme.orbit.colorTextButtonSuccess,
+      [TYPE_OPTIONS.WARNING]: theme.orbit.colorTextButtonWarning,
+      [TYPE_OPTIONS.CRITICAL]: theme.orbit.colorTextButtonCritical,
+      [TYPE_OPTIONS.FACEBOOK]: theme.orbit.colorTextButtonFacebook,
+      [TYPE_OPTIONS.GOOGLE]: theme.orbit.colorTextButtonGoogle,
+    },
+    [TOKENS.colorTextButtonBordered]: {
+      [TYPE_OPTIONS.PRIMARY]: theme.orbit.colorTextButtonPrimaryBordered,
+      [TYPE_OPTIONS.SECONDARY]: theme.orbit.colorTextButtonSecondaryBordered,
+      [TYPE_OPTIONS.INFO]: theme.orbit.colorTextButtonInfoBordered,
+      [TYPE_OPTIONS.SUCCESS]: theme.orbit.colorTextButtonSuccessBordered,
+      [TYPE_OPTIONS.WARNING]: theme.orbit.colorTextButtonWarningBordered,
+      [TYPE_OPTIONS.CRITICAL]: theme.orbit.colorTextButtonCriticalBordered,
+      [TYPE_OPTIONS.FACEBOOK]: theme.orbit.colorTextButtonFacebookBordered,
+      [TYPE_OPTIONS.GOOGLE]: theme.orbit.colorTextButtonGoogleBordered,
+    },
+  };
+  return tokens[name][type];
+};
+
+export const StyledButton = styled(View, props => {
+  const { bordered, theme, block, width, circled, disabled } = props;
+  return {
+    justifyContent: "center",
+    alignItems: "center",
+    height: getSizeToken(TOKENS.heightButton, props),
+    backgroundColor: bordered
+      ? theme.orbit.backgroundButtonBordered
+      : getTypeToken(TOKENS.backgroundButton, props),
+    width: block ? "100%" : width || "auto",
```

Comment:
```suggestion
    width: block ? "100%" : `${width}px` || "auto",
```
as it does not work in `storybook-web` otherwise.